openstack_controller/tests/conftest.py

```diff
@@ -26,3 +27,9 @@ def dd_environment():
             socks_ip, socks_port = socks
             agent_config = {'proxy': {'http': 'socks5://{}:{}'.format(socks_ip, socks_port)}}
             yield instance, agent_config
+
+
+@pytest.fixture(scope='session')
```

Comment:
Since the classes modify headers

```suggestion
@pytest.fixture
```